dlt/destinations/impl/snowflake/configuration.py

```diff
@@ -85,6 +89,13 @@ def to_url(self) -> URL:
             query["warehouse"] = self.warehouse
         if self.role and "role" not in query:
             query["role"] = self.role
+
+        if "application" not in query:
+            if self.application is None:
+                query["application"] = snowflake_application_id
+            elif self.application and self.application != "skip":
```

Comment:
hmmmm you could just allow empty `""` application name and if detected then skip it

Reply:
Got it I will then just use `""` to skip it